examples/customer_web_example.py

```diff
@@ -2,7 +2,7 @@
 
 from livechat.customer.web.base import CustomerWeb
 
-customer_web = CustomerWeb.get_client(license_id=12345,
+customer_web = CustomerWeb.get_client(organization_id='foo-bar-baz',
```

Comment:
OPT Maybe something more similar to org_id instead of strig foo-bar-baz ??
Or even something like in our official docs:
`curl -X GET \
  https://api.livechatinc.com/v3.4/configuration/action/get_license_id?organization_id=142cf3ad-5d54-4cf6-8ce1-3773d14d7f3f`
 It could be more helpful